src/ILVerify/src/ILImporter.Verify.cs

```diff
@@ -1276,7 +1290,15 @@ void ImportStoreIndirect(int token)
 
         void ImportStoreIndirect(TypeDesc type)
         {
-            throw new NotImplementedException($"{nameof(ImportStoreIndirect)} not implemented");
+            ClearPendingPrefix(Prefix.Unaligned);
+            ClearPendingPrefix(Prefix.Volatile);
+
+            var value = Pop();
+            var address = Pop();
+
+            Check(!address.IsReadOnly, VerifierError.ReadOnlyIllegalWrite);
+            CheckLoadIndirect(address, type);
```

Comment:
The name `CheckLoadIndirect` is a little bit misleading here. It simply verifies that the pointer type is compatible with the instruction type (`stind.i1`, `stdind.i4`, ...). The same check is needed for the `ldind.*` instructions. The actual assignability check check is done in the next line.  I'm going to rename this method to `CheckPointerTypeCombatible`.